modules/fluid_properties/test/tests/ics/rho_from_pressure_temperature/tests

```diff
@@ -0,0 +1,8 @@
+[Tests]
+  [test]
+    type = 'CSVDiff'
+    input = 'test.i'
+    csvdiff = 'test_out.csv'
+    rel_err = 1e-10
```

Comment:
I'm sorry I missed this in the first review: we need requirements/design/issues for this and the other specs.

Reply:
yeah, i thought there was a check for this in the pre-check recipe